packages/schematics/angular/migrations/update-8/rules/noLazyModulePathsRule.ts

```diff
@@ -21,7 +21,16 @@ export class Rule extends Rules.AbstractRule {
     const ruleName = this.ruleName;
     const changes: RuleFailure[] = [];
 
-    ts.forEachChild(ast, function analyze(node) {
+    // NOTE: This should ideally be excluded at a higher level to avoid parsing
```

Comment:
What's stopping this from happening? Parsing all of node modules initially sounds pretty bad.

Reply:
Feature additions/fixes would be needed for the tslint fix task used here: https://github.com/angular/angular-cli/blob/master/packages/schematics/angular/migrations/update-8/update-lazy-module-paths.ts#L20

Reply:
An `exclude` option in https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/schematics/tasks/tslint-fix/options.ts#L12 would be good I suppose. We're already using the `include` but that'll match `node_modules` too.